src/sync/installation.ts

```diff
@@ -411,8 +450,13 @@ async function doProcessInstallation(app, queues, job, installationId: number, j
 
 // TODO: type queues
 export const processInstallation =
-	(app: Application, queues) =>
-		async (job, rootLogger: LoggerWithTarget): Promise<void> => {
+	(app: Application, queues) => {
+		const inProgressStorage = new RedisInProgressStorageWithTimeout(new Redis(getRedisInfo("installations-in-progress")));
```

Comment:
I thought that should be incapsulated into the deduplicator

Reply:
Deduplicator is storage-type agnostic.